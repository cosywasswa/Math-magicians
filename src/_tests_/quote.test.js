import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import Quotes from "../components/quotes";

describe("Should render Quotes component", () => {
  test("Should render loading status for quote", () => {
    render(<Quotes />);
    const element = screen.getByTestId("quoteStatus");
    expect(element).toBeTruthy();
  });
  test("Should render according to snapshot", () => {
    const tree = renderer.create(<Quotes />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
