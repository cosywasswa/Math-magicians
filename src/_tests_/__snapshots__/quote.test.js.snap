// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Quotes component Should render according to snapshot 1`] = `[Function]`;
